__device-tests__/gutenberg-editor-sanity-test-1-visual.test.js

```diff
@@ -164,20 +152,11 @@ describe( 'Gutenberg Editor - Test Suite 1', () => {
 			await toggleOrientation( editorPage.driver );
 			// Wait for the device to finish rotating
 			await editorPage.driver.sleep( 3000 );
-			// Navigate upwards in block hierarchy
-			await editorPage.moveBlockSelectionUp();
-			await editorPage.driver.sleep( 1000 );
-			await editorPage.removeBlock();
 		} );
 
 		it( 'displays with correct colors with dark mode enabled', async () => {
 			await toggleDarkMode( editorPage.driver, true );
-
-			// The Android editor requires a restart to apply dark mode
-			if ( isAndroid() ) {
-				await editorPage.driver.resetApp();
-				await isEditorVisible( editorPage.driver );
-			}
```

Comment:
We no longer need to restart the app after reverting the dark mode because next test will launch the app.